frontend/src/components/summary/group/ECEWESummary.vue

```diff
@@ -143,18 +164,15 @@
                         :rules="rules.required"
                       />
                     </v-col>
-                  </v-row>
-                </v-col>
-                <v-col v-else-if="showWageConfirmation" cols="12">
-                  <v-row no-gutters>
-                    <v-col cols="12">
+
+                    <v-col v-if="ecewe?.applicableSector === ECEWE_SECTOR_TYPES.SOME_FACILITIES_UNIONIZED" cols="12">
                       <span class="summary-label pt-3">
-                        I confirm our organization/facilities has reached an agreement with the union to amend the
-                        collective agreement(s) in order to implement the ECE Wage Enhancement.
+                        I confirm our organization/facilities has reached a local agreement with the union to amend the
+                        collective agreement(s) in order to implement the ECE -WE.
```

Comment:
Is this formatting correct? I would either expect spaces on either side or no spaces.

Reply:
you are right it should be no spaces. This is done now